docs/web.rst

```diff
@@ -904,6 +904,10 @@ the **next** *middleware factory*. The last *middleware factory* always receives
 the :ref:`request handler <aiohttp-web-handler>` selected by the router itself
 (by :meth:`UrlDispatcher.resolve`).
 
+.. note::
+
+   The chain of factories is run for every handled request.
```

Comment:
I think someone could still read this and think it's just reiterating that each middleware is called for all views.

Perhaps we could be more explicit, something like 

"Both the outer `middleware_factory` coroutine and the inner `middleware_handler` coroutine are called for every request handled."